src/main/java/com/github/tomakehurst/wiremock/recording/SnapshotStubMappingPostProcessor.java

```diff
@@ -47,30 +49,46 @@ public SnapshotStubMappingPostProcessor(
   }
 
   public List<StubMapping> process(Iterable<StubMapping> stubMappings) {
-    final Multiset<RequestPattern> requestCounts = HashMultiset.create();
-    final List<StubMapping> processedStubMappings = new ArrayList<>();
-
+    // 1. Run any applicable StubMappingTransformers against the stub mappings.
+    ArrayList<StubMapping> transformedStubMappings = new ArrayList<>();
```

Comment:
This plus the next few lines could be done more neatly by mapping over a stream.

Reply:
If we're using `Iterable` as the type for `stubMappings` this actually may be worse from a readability perspective:

```java
List<StubMapping> transformedStubMappings = 
        StreamSupport.stream(stubMappings.spliterator(), false)
                .map(transformerRunner)
                .collect(toList());
```

Note that when using `Iterables` you must use `StreamSupport` to gain access to a `Steam`.

So if you really want to use streams here I'd propose we change the method signature of `process` as well to use `Collection` what you end up with is:

```java
public List<StubMapping> process(Collection<StubMapping> stubMappings) {
    // 1. Run any applicable StubMappingTransformers against the stub mappings.
    List<StubMapping> transformedStubMappings =
            stubMappings.stream().map(transformerRunner).collect(toList());
    ...
```

`Collection` is a bit more narrow than `Iterable` but I think general enough and appropriate for the semantics of `stubMappings`.